x-pack/plugins/observability/public/embeddable/slo/overview/slo_overview.tsx

```diff
@@ -110,52 +87,31 @@ export function SloOverview({
     );
   }
 
-  const TargetCopy = i18n.translate('xpack.observability.sloEmbeddable.overview.sloTargetLabel', {
-    defaultMessage: 'Target',
-  });
-  const extraContent = `${TargetCopy} <b>${numeral(slo?.objective.target).format(
-    percentFormat
-  )}</b>`;
-  // eslint-disable-next-line react/no-danger
-  const extra = <span dangerouslySetInnerHTML={{ __html: extraContent }} />;
-  const metricData =
-    slo !== undefined
-      ? [
-          {
-            color,
-            title: slo.name,
-            subtitle: slo.groupBy !== ALL_VALUE ? `${slo.groupBy}:${slo.instanceId}` : '',
-            icon: getIcon('visGauge'),
-            value:
-              sloStatus === 'NO_DATA'
-                ? NOT_AVAILABLE_LABEL
-                : numeral(slo.summary.sliValue).format(percentFormat),
-            valueFormatter: (value: number) => `${value}%`,
-            extra,
-            trend: [],
-            trendShape: MetricTrendShape.Area,
-          },
-        ]
-      : [];
+  const rules = rulesBySlo?.[slo?.id];
+  const activeAlerts = activeAlertsBySlo.get(slo);
+
+  const hasGroupBy = Boolean(slo.groupBy && slo.groupBy !== ALL_VALUE);
+
+  const historicalSummary = historicalSummaries.find(
+    (histSummary) =>
+      histSummary.sloId === slo.id && histSummary.instanceId === (slo.instanceId ?? ALL_VALUE)
+  )?.data;
+
+  const historicalSliData = formatHistoricalData(historicalSummary, 'sli_value');
+
   return (
-    <>
-      <Chart>
-        <Settings
-          onElementClick={() => {
-            navigateToUrl(
-              basePath.prepend(
-                paths.observability.sloDetails(
-                  slo!.id,
-                  slo?.groupBy !== ALL_VALUE && slo?.instanceId ? slo.instanceId : undefined
-                )
-              )
-            );
-          }}
-          locale={i18n.getLocale()}
+    <div ref={containerRef} style={{ width: '100%' }}>
+      <SloCardChart slo={slo} historicalSliData={historicalSliData ?? []} cardsPerRow={4} />
+      <SloCardBadgesPortal containerRef={containerRef}>
+        <SloCardItemBadges
+          slo={slo}
+          rules={rules}
+          activeAlerts={activeAlerts}
+          handleCreateRule={() => {}}
```

Comment:
do we really need to do that?

Reply:
@shahzad31

Reply:
`SloCardItemBadges` is shared between SLO list page and SLO overview embeddable. In first case it opens a flyout, in second case it shows only a tooltip

<img width="350" alt="Screenshot 2023-12-04 at 12 22 29" src="https://github.com/elastic/kibana/assets/2852703/5f970b03-ae21-402d-9792-0e9a276e10e9">


<img width="350" alt="Screenshot 2023-12-04 at 12 22 06" src="https://github.com/elastic/kibana/assets/2852703/8c7a129a-af0d-430b-ba1a-8cce9c3f19f8">

We could make handleCreateRule optional, but then TS complains about onClick, so I could live with that.

Reply:
if you have a conditional action, just wrap it with an handler, something like that
```ts
const handlerOnclick = useCallback(() => {
  condFunc?.();
}, [condFunc]);

return <div onClick-{handlerOnclick} />
```